### PR TITLE
[RA1] Add biblio for chapter 8

### PR DESCRIPTION
**Pull request by collivier**
Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 1]
```diff
@@ -436,9 +415,8 @@ Image - Glance API testing
 ''''''''''''''''''''''''''
 
 Glance API is covered in the OpenStack Gates via
-`Tempest <https://opendev.org/openstack/tempest>`__ as integrated in
-`Functest Smoke
-CNTT <https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml>`__.
+:cite:p:`rally` as integrated in
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:418]: the original is "Tempest" -- if changing to "rally" then maybe we need a note as to the reason for the change.

File: doc/ref_arch/openstack/chapters/chapter08.rst [hunk 2]
```diff
@@ -436,9 +415,8 @@ Image - Glance API testing
...
+Functest Smoke CNTT :cite:p:`functestsmokecntt`.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.rst:419]: Should CNTT here and elsewhere be changed to Anuket
```suggestion
Functest Smoke Anuket :cite:p:`functestsmokecntt`.
```

**Review by karinesevilla — APPROVED** [review]

File: doc/ref_arch/openstack/chapters/chapter08.rst (hunk 1 above)
Comment (collivier) [reply doc/ref_arch/openstack/chapters/chapter08.rst:418]: @pgoyal01 well catch. cited is wrong here. Thx

Comment (collivier) [reply doc/ref_arch/openstack/chapters/chapter08.rst:418]: Fixed in next commit

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**